spec/order_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'order'
+
+describe Order do
+
+  context "#add_to_basket" do
+    it "should add items to basket" do
+      subject.add_to_basket("Hamburger", 2)
+      expect(subject.basket).to include("Hamburger" => 2)
+    end
+    it "should not add items which aren't on menu" do
+      expect { subject.add_to_basket("Chicken Korma", 4) }.to raise_error "Item not in menu"
```

Comment:
Metrics/LineLength: Line is too long. [92/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)